packages/tabs/src/scss/_component.scss

```diff
@@ -2,218 +2,214 @@
 @use 'global' as *;
 
 // Tabs
-//
-// Variants for tabs
-// .tabs--outline - Rectangle outline style for tab group
-// .tabs--boxed - Rectangle box style for each tab
-// .tabs--square - Square box style for each tab
-// .tabs--two - Style for two tabs
-// .tabs--three - Style for three tabs
-// .tabs--four - Style for four tabs
-// .tabs--five - Style for five tabs
-//
-// Variants for tabs content
-// .tabs-content--container - Rectangle box style for tab content
+.tabs {
 
-// Tabs
-.tabs-section {
-  // Tabs - default
-  .tabs {
+  * {
+    box-sizing: border-box;
+  }
+
+  // Tab list
+  //&__tab-list,
+  [role="tablist"] {
+    position: relative;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+    overflow: visible; 
     padding: core.$space-none;
     margin: core.$space-none;
+  }
+  
+  // Tab Link
+  &__link {
+    display: none;
+
+    @media #{core.$screen-md-up} {
+      display: inline;
+      margin-left: auto;
+      align-self: center;
+    }
+  }
 
-    // Clearfix from Foundation 5
-    &:before, &:after { content: " "; display: table; }
-    &:after { clear: both; }
-
-    // Foundaton classname for li wrapping tab title
-    .tab-title {
-      text-align: center;
-      float: left;
-      list-style: none;
-      margin-bottom: core.$space-none;
-      position: relative;
-      border-bottom: 1px solid $panel-border;
-
-      // active tab
-      &.active {
-        border-bottom: 1px solid $tabs-color;
-        color: $tabs-color;
-      }
-
-      // tab title
-      a {
-        color: $tabs-color;
-        background: core.$uq-white;
-        display: block;
-        padding: core.$space;
-
-        &:hover,
-        &:focus {
-          text-decoration: none;
-        }
-      }
-
-      // tab sub-title
-      .tabs__sub-title {
-        display: block;
-        font-weight: core.$font-weight-normal;
-        font-size: 0.6875rem;
+  // Tab
+  //&__tab,
+  [role="tab"] {
+    position: relative;
+    flex: 1 1 auto;
+    padding: core.$space core.$space-md;
+    margin-right: core.$space-xs;
+    font-size: core.$font-size-m;
+    font-weight: core.$font-weight-thin;
+    color: core.$uq-neutral-dark80;
+    background-color: core.$uq-white;
+    border-radius: core.$space-xs core.$space-xs core.$space-none core.$space-none;
+    border: none;
+    text-align: center;
+    cursor: pointer;
+
+    @media #{core.$screen-md-up} {
+      flex: 0 1 auto;
+      padding: core.$space-md core.$space-lg;
+      margin-right: core.$space-xs;
+      font-size: core.$font-size-l;
+    }
 
-        @media #{core.$screen-md-up} {
-          font-size: core.$font-size-s;
-        }
-      }
+    &:last-child {
+      margin-right: core.$space-none;
     }
 
-    // ***********************************
-    // Tabs - fixed
-    &--fixed {
-      position: fixed;
-      top: 0;
+    &:before {
```

Comment:
Pseudo-elements should be written using a double colon (::). E.g. `.some-class::before, .some-class::after etc.` Some reading: https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements. @Bcrunk do you agree?